tasks/coffee-shop/coffee-shop.md

```diff
@@ -0,0 +1,112 @@
+  - In this part of the task, it is necessary to add responsiveness to the pages created in the first stage, up to and including a width of 380px;
+  - The validity of the work is checked, its alignment with the layout at the corresponding page width, layout responsiveness, and the absence of horizontal scroll bars.
+- [Coffee-shop. Week-3: Adding functionality](coffee-shop-week3.md)
+  - In this part of the task, we use JavaScript to add interactivity to the pages;
+  - The implemented functionality is being checked.
+
+The duration of each part of the task is 1 week.  
+The form of evaluation for each part of the task is a cross-check review.
+
+[Layout in Figma](#)
+
+## Creating your own copy of the layout
+Start the task by creating your own copy of the layout. To do this:
+- Log in to [Figma](https://www.figma.com/);
+- Open the layout;
+- On the top panel, click on the arrow next to the layout name, and select the option "Duplicate to your drafts";
+- At the top left, open the settings, and choose "Back to files";
+- Open the copy of the layout with the label "In Drafts".
+
+## Code quality recommendations
+Recommendations are provided for reference; strict adherence to them is not expected and will not be checked.
+- [Guideline](https://codeguide.academy/html-css.html#html)
+- Code quality guide
+  - [General principles](../../stage1/modules/clean-code/materials/generic-principles.md)
+  - [HTML and CSS recommendations - beginner level](../../stage1/modules/clean-code/materials/html-and-css.md)
+  - [HTML and CSS recommendations - advanced level](../../stage1/modules/clean-code/materials/html-and-css-extended.md)
+
+## Technical requirements
+1. The layout is valid, semantic, and matches the design.
+2. The application is displayed correctly and functions properly in the latest version of Google Chrome.
+3. Using CSS frameworks (e.g., `Bootstrap`) **is not allowed**.
+4. Using JS frameworks (e.g., `Angular`, `React`, `Vue`, etc.) **is not allowed**.
+5. Using outdated libraries (e.g. `JQuery`, etc.) or pre-built libraries (e.g. `Swiper`, etc) to implement functionality **is not allowed**.
+6. Using `TypeScript` **is not allowed**.
+7. Using CSS preprocessors (`SASS`, `SCSS`), `normalize.css` is allowed.
+8. Using a style reset with `reset.css` is not recommended.
+9. Adding layout as an image by taking a screenshot of a part of the layout and pasting it into the markup is not allowed. Please use tags and characters for layout, and use images only for adding pictures and icons, not for layout elements (buttons, blocks, sections).
+10. The code must be readable, without minification or obfuscation.
+
+## Repository Requirements
+- Task should be done in private school's repository. [How to work with a school's private repository](https://docs.rs.school/#/private-repository)
+- Create new branch `coffee-shop` from `main`. Create folder `coffee-shop` in the created branch where you will do source code
```

Comment:
Create a folder coffee-shop in the created branch. Your code should be in this folder. || Place your code in this folder || Write your code in this folder.